src/main/java/com/EighthLight/app/IStrategy.java

```diff
@@ -1,5 +1,5 @@
 package com.EighthLight.app;
 
 public interface IStrategy {
-    void markBoard(IBoard board, String aiSymbol, String humanSymbol);
+    void markBoard(IBoard board);
```

Comment:
IStrategy interface provides the "contract" for all difficultly methods to follow. All difficulties must have a markBoard method that takes a board. The constructors in each difficulty provide information unique to that instance.